lambdas/ttl-handle-expiry-lambda/src/apis/dynamodb-stream-handler.ts

```diff
@@ -80,6 +76,8 @@ export const createHandler = ({
         return;
       }
 
+      const itemEvent: MESHInboxMessageDownloaded = item.event as any;
```

Comment:
An option to avoid the `as any` cast. I try and avoid them as much as possible outside of unit tests.
```suggestion
      let itemEvent: MESHInboxMessageDownloaded;

      if ((messageDownloadedValidator as (d: unknown) => d is MESHInboxMessageDownloaded)(item.event)) {
        itemEvent = item.event;
      } else {
        logger.warn({
          err: messageDownloadedValidator.errors,
          description: 'Error parsing ttl item event',
        });

        failures.push(record);

        return;
      }
```

Reply:
Updated.